include/pybind11/detail/exception_translation.h

```diff
@@ -0,0 +1,71 @@
+/*
+    pybind11/detail/exception_translation.h: means to translate C++ exceptions to Python exception
+
+    Copyright (c) 2024-2024 The Pybind Development Team.
```

Comment:
Just `2024`?

Also maybe an extra `s` at the end of the line above? (... translate ... excpetions to ... exceptions)

Reply:
fixed